README.md

```diff
@@ -100,6 +100,52 @@ You are free to implement the genome classes however you want, and using whateer
 
 When you have implemented the two (or more) classes, describe the complexity of each operation as a function of the genome size (at the time of the operation), and the size of the TE involved (and when copying, the offset you are copying). Put the description here:
 
-**FIXME: OPERATION COMPLEXITY**
+# Awnser:
+
+*ListGenome:*
+
+init = O(n), creating a list of length n.
+
+insert = O(te + n), creating a list of size te + inserting this list + (correcting).. probably something like te^2 * n i practise. 
+
+copy = O(n + te), uses insert so same as above.. 
+
+disable = O(te), removes from dict O(1) + corrects te characters. 
+
+active te's = O(1), dict.keys() is O(1) in python3.
```

Comment:
It's an iterator that you can get in constant time, but to put it in a list (with the `list()` function) takes time $k$ if there are $k$ elements.